test/golden/EraseType.hs

```diff
@@ -16,5 +16,4 @@ testCong :: Int
 testCong = 1 + testRezz
 
 rTail :: [Int] -> [Int]
-rTail = \ ys -> tail ys
-
+rTail ys = tail ys
```

Comment:
We should not apply the linting to the files in the `golden` directory, since these are the expected outputs of the test suite. However, we could consider applying the auto-formatter to the code generated by `agda2hs` (which would then lead to an update of the golden files too).